app/.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_7" project-jdk-name="11.0.6" project-jdk-type="JavaSDK">
```

Comment:
@dojodj has linux, that's why we went with 11. Maybe this doesn't make a difference though, but what is "1.8"?